_episodes/40.md

```diff
@@ -0,0 +1,425 @@
+---
+layout: episode
+title:  "40: Trino's cold as ice!"
+date: 2022-09-07
+tags: iceberg apache datalake table format sql schema evolution partition optimize
+youtube_id: ""
+wistia_id: ""
+sections: 
+   - title: "Concept of the week"
+     desc: "Latest features in Apache Iceberg and the Iceberg connector"
+     time: 0
+   - title: "PR of the week"
+     desc: ""
+     time: 0
+   - title: "Demo"
+     desc: "Merge tables with Iceberg and reading the data in Trino"
+     time: 0
+   - title: "Question of the week"
+     desc: ""
+     time: 0
+---
+
+<p align="center">
+<img align="center" width="50%" height="100%" src="/assets/episode/15/trino-iceberg.png"/><br/>
+Looks like Commander Bun Bun is safe on this Iceberg<br/>
+<a href="https://joshdata.me/iceberger.html">https://joshdata.me/iceberger.html</a>
+</p>
+
+## Guests
+ * Ryan Blue, creator of Iceberg and CEO at 
+ [Tabular](https://tabular.io) ([@rdblue](https://github.com/rdblue))
+ * Sam Redai, Developer Advocate at [Tabular](https://tabular.io)
+ ([@samuelredai](https://twitter.com/samuelredai))
+
+## Releases 394 to 395
+
+Official highlights from Martin Traverso:
+
+[Trino 394]({{site.url}}/docs/current/release/release-394.html)
+
+* 
+
+[Trino 395]({{site.url}}/docs/current/release/release-395.html)
+
+* 
+
+Additional highlights worth a mention according to Cole:
+
+* 
```

Comment:
```suggestion
* The improved performance of inserts on Delta Lake, Hive, and Iceberg is a huge one. We're not entirely sure how much it'll matter in production use cases, but some of the benchmarks suggested it could be massive - one test showed a 75% reduction in query duration.
* Dynamic function resolution in SPI is going to unlock some very neat possibilities down the line.
```